service/src/main/kotlin/fi/espoo/evaka/application/ApplicationQueries.kt

```diff
@@ -1477,8 +1477,10 @@ SELECT
     FALSE AS connected_daycare_only
 FROM application a
 JOIN placement p ON a.child_id = p.child_id AND daterange(p.start_date, p.end_date, '[]') @> ${bind(today)}
+JOIN placement next_p ON a.child_id = next_p.child_id AND next_p.start_date > p.end_date AND next_p.unit_id != p.unit_id
```

Comment:
Tää voi joinata useaan. Tarvii ehkä lateral join jossa order by start_date limit 1. Tosin jos siitä sijoituksesta ei käytetä mitään vaan riittää että niitä on vähintään yksi tulevaisuudessa niin sitten helpoin lisätä WHERE-lauseeseen AND EXISTS(...)

Reply:
ehkä joku toSet jossain poistaa jo duplikaatit mutta siistimpää kuitenkin

Reply:
Siirsin samalla `decision` taulun joinin where-puolelle EXISTS-tyyliin. Ehkä hieman loogisempi niin.